scss/components/_nav.scss

```diff
@@ -13,3 +13,29 @@
     }
   }
 }
+
+.navbar-brand {
+  color: $secondary;
+  &:before {
+    font-size: calc-em(35,20);
+    @include media-breakpoint-up(lg) {
+      font-size: calc-em(44,20);
+    }
+  }
+
+  &:hover {
+    color: $secondary;
+  }
+}
+
+.announcement-bar {
```

Comment:
annoucement-bar has its own component and should have its own scss as well, even if it's only 1 line